src/ralph/backends/data/clickhouse.py

```diff
@@ -235,6 +233,7 @@ def read(
             chunk_size = self.default_chunk_size
 
         query = (
+            # pylint: disable=too-many-function-args
```

Comment:
```suggestion
```
Not sure to see the purpose of disabling this warning here (I can't reproduce it)

Reply:
Interesting, I couldn't get it to pass pylint locally without it. I'll remove it and see what happens in CI.

Reply:
Looks like it works without it! Maybe because I was running Python 3.8 locally and CI is at 3.11. 🤷